doc/dev/developer_guide/tests-unit-tests.rst

```diff
@@ -76,35 +75,43 @@ the `cram task`_.
 .. _`cram`: https://bitheap.org/cram/
 .. _`cram task`: https://github.com/ceph/ceph/blob/master/qa/tasks/cram.py
 
-Tox based testing of python modules
+Tox based testing of Python modules
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some of the Python modules in Ceph use `tox <https://tox.readthedocs.io/en/latest/>`_ to run their unit tests.
 
-Most python modules can be found under ``./src/pybind/``.
+Most of these Python modules can be found in the directory ``./src/pybind/``.
 
-Many modules use **tox** to run their unit tests.
-**tox** itself is a generic virtualenv management and test command line tool.
+Currently the following modules use tox:
 
-To find out quickly if tox can be run you can either just try to run ``tox``
-or find out if a ``tox.ini`` exists.
+* Cephadm (``./src/pybind/mgr/cephadm``)
+* Insights (``./src/pybind/mgr/insights``)
+* Manager core (``./src/pybind/mgr``)
+* Dashboard (``./src/pybind/mgr/dashboard``)
+* Python common (``./src/python-common/tox.ini``)
 
-Currently the following modules use tox:
+Configuring Environments and Tasks in Tox
```

Comment:
Please change this heading to:

```
Configuring Tox environments and tasks
```

This is because we do not use U.S.-style initial caps for headings in this document. In other words, if there was a chapter or section called "Foo bar blaz", the heading would be "Foo bar blaz", and not "Foo Bar Blaz".